.github/workflows/ci.yaml

```diff
@@ -220,7 +220,7 @@ jobs:
             SANDBOX_ID="default"
             CHART_REF="$GITHUB_SHA"
             KUBERNETES_ENV="staging"
-            REPLICA_COUNT="2"
+            REPLICA_COUNT="1"
```

Comment:
we'll have to wait a bit of extra time when running the pipeline from gem2s (if it's slower than 3/4 minutes) but should be ok, we can decrease and see if it bothers people

Reply:
Let's try with 1 and see if it's ok. If it isn't, we will increase it back to 2.